src/engraving/dom/edit.cpp

```diff
@@ -6039,359 +6046,353 @@ void Score::undoAddElement(EngravingItem* element, bool addToLinkedStaves, bool
             ElementType::PEDAL,
             ElementType::LYRICS
         };
-        voice_idx_t voice = track2voice(strack);
-        if (staff->isVoiceVisible(voice) && linkedTracks.empty() && muse::contains(VOICE1_COPY_TYPES, et)) {
```

Comment:
It looks like `VOICE1_COPY_TYPES` is now unused. The comment above its definition suggests that it might have been important. Are you sure it's not necessary anymore?

Reply:
Oops, good catch, thanks